src/libsync/theme.h

```diff
@@ -256,7 +256,6 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
      * it has a trailing slash, for example "remote.php/webdav/".
      */
     virtual QString webDavPath() const;
-    virtual QString webDavPathNonShib() const;
```

Comment:
we might need to touch ownbrander.

Reply:
https://github.com/owncloud/ownbrander/search?q=webDavPathNonShib&unscoped_q=webDavPathNonShib

```We couldn’t find any code matching 'webDavPathNonShib' in owncloud/ownbrander```